src/chocolatey/GetChocolatey.cs

```diff
@@ -49,8 +49,9 @@ public class GetChocolatey
 
         public GetChocolatey()
         {
+            Log4NetAppenderConfiguration.configure();

```

Comment:
I'm also not sure it actually sets the configuration active though. You would need a call to https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure/logging/Log4NetLog.cs#L18 to activate the logger.